docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md

```diff
@@ -0,0 +1,281 @@
+
+To use Hasura with YugabyteDB, the configuration should be similar to PostgreSQL, except that the port should be `5433`.
+
+## Setting Up the Benchmark
+
+You can use a [benchmark tool](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/graphql-subscription-perf-tool) to deploy the benchmark setup on a Kubernetes cluster. The entire setup runs inside Kubernetes and includes the following components: 
+
+- A YugabyteDB cluster running natively inside Kubernetes.
+- A Hasura Kubernetes service.
+- A [sample application](#sample-application) consisting of a load generator that produces user orders and a GraphQL application that subscribes to incoming orders.
+
+The following diagram illustrates the setup:
+
+![Benchmark](/images/develop/ecosystem-integrations/hasura-benchmark.png)
+
+### Sample Application
+
+The sample application provides a simulation of users placing orders that require delivery. Each order is tracked and a notification is posted in real time to trigger the order fulfillment.
+
+The following `user_account` table contains information about the user:
+
+```sql
+CREATE TABLE user_account (
+	userID BIGINT NOT NULL,
+	accountName VARCHAR,
+	givenName VARCHAR,
+	middleName VARCHAR,
+	familyName VARCHAR,
+	userGender VARCHAR,
+	userAge INT,
+	dob TIMESTAMP,
+	address1 VARCHAR,
+	address2 VARCHAR,
+	city VARCHAR,
+	zip VARCHAR,
+	email VARCHAR,
+	homePhone VARCHAR,
+	mobilePhone VARCHAR,
+	country VARCHAR,
+	company VARCHAR,
+	companyEmail VARCHAR,
+	active BOOLEAN,
+	PRIMARY KEY (userID HASH)
+);
+
+CREATE INDEX user_fname ON user_account (givenName) ;
+CREATE INDEX user_lname ON user_account (familyName) ;
+```
+
+The following `user_orders` table contains a list of orders placed by the user:
+
+```sql
+CREATE TABLE user_orders (
+	userID BIGINT NOT NULL ,
+	orderID VARCHAR NOT NULL ,
+	orderTotal VARCHAR NOT NULL ,
+	orderDetails VARCHAR NOT NULL,
+	orderTime TIMESTAMP NOT NULL,
+	PRIMARY KEY (userID HASH, orderID ASC)
+);
+
+ALTER TABLE user_orders ADD FOREIGN KEY (userID) REFERENCES user_account(userID);
+```
+
+The following `events` table is used for verifying that all subscriptions are received by the GraphQL tool:
+
+```sql
+CREATE TABLE events (
+	label VARCHAR NOT NULL,
+	connection_id INT NOT NULL,
+	operation_id INT NOT NULL,
+	event_number INT NOT NULL,
+	event_data JSONB NOT NULL,
+	event_time TIMESTAMP NOT NULL,
+	is_error BOOLEAN NOT NULL,
+	latency INT,
+	PRIMARY KEY (connection_id HASH, label, operation_id, event_number)
+);
+```
+
+The following is the dominant query performed using subscriptions in Hasura. This query produces a list of the most recent orders placed by a given user:
+
+```
+subscription ($id: bigint!) {
+ user_account (where: {userid: {_eq: $id}}) {
+   accountname
+   givenname
+   familyname
+   city
+   user_orders(order_by: {orderid: desc}) {
+     orderid
+     ordertotal
+     orderdetails
+     ordertime
+     deliveryStatus
```

Comment:
We have to remove `deliveryStatus` attribute, currently, it's not being used.

Reply:
Deleted